### ISSUE TITLE
Autofix is not working in multiline cases

### ISSUE THREAD

**Issue by Andi-Lo** [jerone/eslint-plugin-angular-template-consistent-this#1]
Hi,

Exactly the rule I was looking for  👍  and was about to write myself (but struggled quite a bit with it)

I found an issue tho with the auto-fixing when having multiline statements (using the implicit option)

```javascript
      <button
        [disabled]="
          !(myCondition.fooBarBaz.zapZapZap && myCondition.fooBarBaz.zipZapZup) &&
          this.implicitVar === 'something'
        "
      ></button>
```

This won't get fixed correctly and also the wrong property is underlined in my code editor: `implicitVar` instead of `this`

**Comment by jerone** [comment]
Correct. I didn't complete the plugin yet. It's kind of blocking on angular-eslint/angular-eslint#384

**Comment by jerone** [comment]
It looks like I found an solution in fb7c29c. Will release a new version soon.

**Comment by jerone** [comment]
Lot's of issues where fixed in version 0.1.0, but still some left, based on my own findings:

---

```html
					<tr
						*ngIf="
							currentMutation.address.opmerking !== null &&
							currentMutation.address.opmerking.length > 0
						">
```

![](https://user-images.githubusercontent.com/55841/179350438-420b94a6-84d8-4c24-bb65-d1082790e120.png)

Potential issue: multi-line `*ngIf`.

---

```typescript
@Component({
	template: `
		<form clrStepper [formGroup]="form">
			<clr-stepper-panel formGroupName="group">
				<button clrStepButton="next" navigateToNextStep="true">Next</button>
			</clr-stepper-panel>
		</form>
	`
})
class TestComponent {
	@ViewChild(ClrStepButton, { static: false }) public button: ClrStepButton;

	public form = new FormGroup({ group: new FormGroup({}) });
}
```

![](https://user-images.githubusercontent.com/55841/179350476-87520416-7578-4dff-8a30-880d167f6867.png)

Which resolves to incorrect HTML with `this.` before the double quote:

![](https://user-images.githubusercontent.com/55841/179350530-36223e42-4fbb-4ad3-9dbf-1c7264629662.png)

**Comment by jerone** [comment]
More use-cases have been patched in version 0.2.0.

Aldo, all tests are green, after running `ng lint --fix` in my project, not all cases are fixed correctly (specifically see comment above).

Fixing those edge-cases manually, they don't report.

Side-note: I also noticed differences in lint fixing when running `ng lint --fix` and saving file with `settings.json`:

```jsonc
	"editor.formatOnPaste": true,
	"editor.formatOnSave": true,
	"editor.formatOnType": true,
	"editor.codeActionsOnSave": {
		"source.fixAll.eslint": true // Fix all fixable ESLint rules.
	},
```

**Comment by jerone** [comment]
I'll do some more tests, but for now, I'm marking this issue as fixed.
